src/golang/lib/collections/shared/types.go

```diff
@@ -30,6 +30,14 @@ const (
 	UnknownExecutionStatus    ExecutionStatus = "unknown"
 )
 
+type NotificationLevel string
```

Comment:
Can you use the constants defined [here] (https://github.com/aqueducthq/aqueduct/blob/main/src/golang/lib/models/shared/notification.go#L10) instead of redefining them. I'd like us to deprecate `lib/collections`.